pom.xml

```diff
@@ -7,7 +7,7 @@
   <groupId>ir.sahab</groupId>
   <artifactId>spring-boot-rest-commons</artifactId>
   <name>Spring Boot Rest Commons</name>
-  <version>1.3.0</version>
+  <version>1.3.1</version>
```

Comment:
It is not a bug fix. If we want to be very strict, it is a non-backward compatible change but we can accept it as a very low effect change. I prefer the second and I suggest 1.4.0